nxc/protocols/smb.py

```diff
@@ -549,7 +549,6 @@ def create_smbv1_conn(self):
                 preferredDialect=SMB_DIALECT,
                 timeout=self.args.smb_timeout,
             )
-            self.smbv1 = True
```

Comment:
no need to remove

Reply:
The reason why I remove it, because I saw https://github.com/Pennyw0rth/NetExec/blob/b79ddec91f9f712841049c1c7453322d2c1682c3/nxc/protocols/smb.py#L596 is always do `self.smbv1 = self.create_smbv1_conn()`, so here the `self.smbv1=True` is no needed

Reply:
good catch !